OpenLaTeXStudio/EditorModule/src/latexstudio/editor/toolbar/ToggleComment.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package latexstudio.editor.toolbar;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import latexstudio.editor.EditorTopComponent;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.NbBundle.Messages;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ *
+ * @author Labelpm
+ */
+@ActionID(
+        category = "Edit",
+        id = "latexstudio.editor.ToggleComment"
+)
+@ActionRegistration(
+        iconBase = "latexstudio/editor/resources/icons/comment-symbol24.png",

```

Comment:
We should add a 16x16px version and link to it here, naming it "comment-symbol.png". The 24x24 version will be discovered automatically based on the name, by the NetBeans Platform.